public/scripts/cookie-consent-notice.js

```diff
@@ -0,0 +1,66 @@
+(function (c, d, o, g) {
```

Comment:
@SovereignAndrey, I tried to put script content into a module for uniformity with the other scripts. Kindly revert if this has bad effect (though tested still works). Overall, PR looks good regardless

Reply:
@ap-justin Good call, much cleaner! :heart: